assert.throws(TypeError, function() {
  0, [] = 1;
});